src/main/java/com/qaprosoft/carina/demo/gui/components/Header.java

```diff
@@ -57,7 +70,75 @@ public void verifyHeaderBasicElements(SoftAssert softAssert) {
         softAssert.assertTrue(instagramButton.isElementPresent(), "Instagram button was not found!");
         softAssert.assertTrue(youTubeButton.isElementPresent(), "YouTube button was not found!");
         softAssert.assertTrue(rssButton.isElementPresent(), "RSS button was not found!");
-        softAssert.assertTrue(logInButton.isElementPresent(), "Log in button was not found!");
+        softAssert.assertTrue(logInIcon.isElementPresent(), "Log in button was not found!");
         softAssert.assertTrue(signUpButton.isElementPresent(), "Sign up button was not found!");
     }
+
+    public void openFieldsForLogIn() {
+        logInIcon.click();
+    }
+
+    public void typeEmail(String email) {
+        emailLabel.type(email);
+    }
+
+    public void typePassword(String password) {
+        passwordLabel.type(password);
+    }
+
+    public void logInGood() {
+        String email = "s9rowa@mail.ru";
+        String password = "changeme";
+
+        openFieldsForLogIn();
+        typeEmail(email);
+        typePassword(password);
+        logInButton.click();
+    }
+
+    public boolean isLogOutIconButtonPresent() {
+        return logOutIconButton.isElementPresent();
+    }
+
+    public FailedLoginPage logInFailedByEmail() {
```

Comment:
This methods(logInFailedByEmail, logInFailedByPassword, logInByWrongFormatEmail), almost the same.How about create  one method login() for example?